lib/internal/core.js

```diff
@@ -10,7 +10,7 @@
  *  Description :
  *
  */
-var OAuthSimple = require('oauthsimple');
+var OAuthSimple = require('../../node_modules/oauthsimple/js/OAuthSimple.js');
```

Comment:
:-1: this could get real messy if this file (`lib/internal/core.js`) ever moves...